aries-site/src/layouts/content/Example/Container.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box, defaultProps } from 'grommet';
+
+// This creates the overall canvas/frame that the Example sits inside of
+export const Container = ({
+  designer,
+  docs,
+  figma,
+  height: heightProp,
+  horizontalLayout,
+  plain,
+  screenContainer,
+  showResponsiveControls,
+  template,
+  ...rest
+}) => {
+  // Height for template screen needs to be between medium and large
+  // to maintain aspect ratio, so this is small + medium
+  const { small, medium } = defaultProps.theme.global.size;
+  const aspectHeight = `${parseInt(medium, 10) + parseInt(small, 10)}px`;
```

Comment:
I wonder if this is more straight forward?
```suggestion
  const aspectHeight = `${Number(medium) + Number(small)}px`;
```